drivers/flash/nrf_qspi_nor.c

```diff
@@ -661,6 +661,10 @@ static int qspi_nrfx_configure(const struct device *dev)
 	nrfx_err_t res = nrfx_qspi_init(&QSPIconfig, qspi_handler, dev_data);
 	int ret = qspi_get_zephyr_ret_code(res);
 
+	if (ret == 0 && !nrf52_errata_121()) {
```

Comment:
This should be `nrf53_errata_121()` to check if a given device is affected by [this anomaly](https://infocenter.nordicsemi.com/topic/errata_nRF5340_Rev1/ERR/nRF5340/Rev1/latest/anomaly_340_121.html?cp=3_0_1_0_1_16). Anomaly 121 in nRF52 Series is [something different](https://infocenter.nordicsemi.com/topic/errata_nRF52840_EngA/ERR/nRF52840/EngineeringA/latest/anomaly_840_121.html?cp=4_0_1_5_1_28).

Reply:
Uh... Good catch. I'm not used to nRF53 being a thing. Whoops!